### ISSUE TITLE
[TW-521] Error in output of "task projects"

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#556]
_Ingo Jauer on 2012-04-26T09:22:35Z says:_ 

 The header of the output of the command "task projects" contains an error. The columns of "Pri:H" and "Pri:L" are interchanged. You can test this with following commands:

$ task add project:prioH priority:H Task with high priority

$ task add project:prioM priority:M Task with medium priority

$ task add project:prioL priority:L Task with low priority

$ task projects

Project Tasks Pri:None Pri:H Pri:M Pri:L
* ------ ----- -------- ----- ----- -----
prioH       1        0     0     0     1
prioL       1        0     1     0     0
prioM       1        0     0     1     0

3 projects (3 tasks)

I compiled the program by myself. This is the output of "task diag":

task 2.0.0
  Platform: Linux

Compiler
   Version: 4.4.5
      Caps: +stdc +stdc_hosted +200809 +LP64 +c1 +i4 +l8 +vp8

Libraries
       Lua: n/a

Build Features
     Built: Apr 24 2012 17:48:10
    Commit: 03799eb
     CMake: 2.8.2
      Caps: -pthreads +srandom +random -uuid

Configuration
      File: /home/ij/.taskrc (found), 1266 bytes, mode 100644
      Data: /home/ij/.task (found), dir, mode 40755
    Server: 
   Locking: Enabled

External Utilities
       scp: (found)
     rsync: version 3.0.7
      curl: curl 7.21.0

Tests
  UUID gen: 1000 unique UUIDs generated.
     $TERM: xterm (144x64)
      Dups: Scanned 3 tasks for duplicate UUIDs:
            No duplicates found

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2012-04-26T09:22:35Z
Modified: 2014-02-09T02:04:40Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2012-04-26T11:10:02Z says:_ 

 Thank you Ingo, this was nicely reported.

You'll be glad to know this was recently fixed, (see #964).
